<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>563</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QTextEdit" name="textEdit_13"/>
     </item>
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEdit_00">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QTextEdit" name="textEdit_04"/>
     </item>
     <item row="2" column="2">
      <widget class="QTextEdit" name="textEdit_22"/>
     </item>
     <item row="1" column="4">
      <widget class="QTextEdit" name="textEdit_14"/>
     </item>
     <item row="3" column="2">
      <widget class="QTextEdit" name="textEdit_32"/>
     </item>
     <item row="4" column="2">
      <widget class="QTextEdit" name="textEdit_42"/>
     </item>
     <item row="5" column="2">
      <widget class="QTextEdit" name="textEdit_52"/>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="textEdit_01"/>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="textEdit_10"/>
     </item>
     <item row="2" column="0">
      <widget class="QTextEdit" name="textEdit_20"/>
     </item>
     <item row="4" column="0">
      <widget class="QTextEdit" name="textEdit_40"/>
     </item>
     <item row="3" column="0">
      <widget class="QTextEdit" name="textEdit_30"/>
     </item>
     <item row="5" column="0">
      <widget class="QTextEdit" name="textEdit_50"/>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="textEdit_11"/>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="textEdit_31"/>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="textEdit_21"/>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="textEdit_51"/>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="textEdit_41"/>
     </item>
     <item row="3" column="3">
      <widget class="QTextEdit" name="textEdit_33"/>
     </item>
     <item row="2" column="4">
      <widget class="QTextEdit" name="textEdit_24"/>
     </item>
     <item row="3" column="4">
      <widget class="QTextEdit" name="textEdit_34"/>
     </item>
     <item row="2" column="3">
      <widget class="QTextEdit" name="textEdit_23"/>
     </item>
     <item row="5" column="4">
      <widget class="QTextEdit" name="textEdit_54"/>
     </item>
     <item row="4" column="4">
      <widget class="QTextEdit" name="textEdit_44"/>
     </item>
     <item row="4" column="3">
      <widget class="QTextEdit" name="textEdit_43"/>
     </item>
     <item row="5" column="3">
      <widget class="QTextEdit" name="textEdit_53"/>
     </item>
     <item row="0" column="3">
      <widget class="QTextEdit" name="textEdit_03"/>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="textEdit_12"/>
     </item>
     <item row="0" column="2">
      <widget class="QTextEdit" name="textEdit_02"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
